Packs/GDPR/pack_metadata.json

```diff
@@ -11,6 +11,8 @@
         "Case Management"
     ],
     "tags": [],
```

Comment:
tags: GDPR

Reply:
I didn't add as tag because searching for GDPR would bring this up anyway, but we could in case we have more of those